zerver/openapi/zulip.yaml

```diff
@@ -8585,6 +8585,14 @@ paths:
                           (/help/invite-new-users#change-who-can-send-invitations) to the organization.
 
                           **Changes**: New in Zulip 4.0 (feature level 51).
+                      can_move_messages_between_streams:
+                        type: boolean
+                        description: |
+                          Present if `realm_user` is present in `fetch_event_types`.
+
+                          Whether the current user is allowed to move messages from one stream to other.
+
+                          **Changes**: New in Zulip 4.0 (feature level 56).
```

Comment:
I think my preference after thinking about this is that we should stop adding `can_` elements to the API -- they should instead be functions that have a copy in `settings_config.js` or `settings_data.js` -- precisely because we want clients to be able to live-update for this sort of thing.